my-app/src/Pages/Home.tsx

```diff
@@ -35,6 +36,26 @@ const Home: React.FC<{}> = (props: any) => {
           placeholder="Enter your search term here"
         />
       </div>
+      <select onChange={(e) => setSelectedGenre(e.target.value)}>
+        <option value="" label="Select a genre">
+          Select a genre
+        </option>
+        <option value="action" label="action">
+          Action
+        </option>
+        <option value="drama" label="drama">
+          Drama
+        </option>
+        <option value="documentary" label="documentary">
+          Documentary
+        </option>
+        <option value="horror" label="horror">
+          Horror
+        </option>
+        <option value="comedy" label="comedy">
+          Comedy
+        </option>
```

Comment:
Label things with capital, so that you are consistent with the labels that you have at the createMovie form